### ISSUE TITLE
Could you provide the pretraining weights

### ISSUE THREAD

**Issue by RefDawn-XD** [IBM/AFDistill#1]
Hi,
    Thanks for your wonderful work, I just wondering if you could provide the training weights? I want to have a try based on your model. Thanks a lot!

**Comment by zackmawaldi** [comment]
Yes please! I can't seem to find the pre-trained weights for plddt and TM models... it would make it so much easier to use it! Thanks!

**Comment by chaconlab** [comment]
Please, could you include the pre-trained weights?

**Comment by PraljakReps** [comment]
Where are the pretrained weights?

**Comment by PraljakReps** [comment]
I see that the paper has been published in a peer-reviewed article (https://www.nature.com/articles/s41598-025-00436-1); however, do we have the pretrained weights available yet?

**Comment by ajasja** [comment]
I agree, without the pretrained weight this repo is much less useful. @imelnyk
